### ISSUE TITLE
virtio_net: Could not create vmnet interface, permission denied or no entitlement?

### ISSUE THREAD

**Issue by rjocoleman** [nlf/dlite#179]
I've been unable to connect to Docker on today's master of dlite and dhyve-os as well as dlite 2.0.0-beta5 with dhyve-os v3.0.0-beta4 (with the docker binary download fixes backported).

Upon checking the logs I've got the `virtio_net: Could not create vmnet interface, permission denied or no entitlement?` thing happening.

I didn't have `/Library/Preferences/SystemConfiguration/com.apple.vmnet.plist`
Running `El Capitan` 10.11.4, upgraded from Yosemite and Mavericks. I've been using other `Hypervisor.framework` based tools but not xhyve (outside of dlite).

I followed this issue (https://github.com/nlf/dhyve/issues/37) and ran https://github.com/ailispaw/boot2docker-xhyve one time this created the file.

To get up and running I then ran (maybe this could have consolidated):

``` shell
$ dlite stop
$ sudo dlite uninstall
$ sudo dlite install
$ dlite start
```

**Comment by nlf** [comment]
and it ended with things working for you?

**Comment by rjocoleman** [comment]
yes 😊

**Comment by nlf** [comment]
so i'm assuming it's safe to close this issue since most of the things you brought up are known. thanks for the feedback!

**Comment by rjocoleman** [comment]
sure, I figured I'd document this here as it wasn't in the issue tracker for this project and will be an issue for some when v2 drops in earnest. 

I'm going to spend some time messing about with a fresh install of El Capitain to see if I can reproduce a minimum fix without going to the level of downloading `ailispaw/boot2docker-xhyve`.
I'm thinking this is just running xhyve with the correct params.. but once I've got a reliable reproduction it reveal a be suitable fix here or in `TheNewNormal/libxhyve`.

I'll report back.
